### ISSUE TITLE
Check results

### ISSUE THREAD

**Issue by AnHeuermann** [JKRT/OMBackend.jl#14]
We should compare results with some provided csv / mat file.

**Comment by SVAGEN26** [comment]
Closing as stale. This is a generic 'compare results with csv/mat' wish from the earliest days of the project; the testsuite has matured well past this and there is no actionable scope left on the original ticket.

**Comment by SVAGEN26** [comment]
@JKRT please close: stale, no actionable scope; testsuite has matured well past 'compare results with csv/mat'.

**Comment by JKRT** [comment]
Yes and it is done in OMLibraryTesting.jl now including with reference CSV files
